test/backend/fused_op.in.cpp

```diff
@@ -1149,49 +1149,724 @@ NGRAPH_TEST(${BACKEND_NAME}, gemm_broadcast_axes_1_input_C)
-                                -1.0,
-                                0.0,
-                                1.0,
-                                9.99999,
-                                10.0,
-                                10.000001,
-                                15.0,
-                                19.999999,
-                                20.0,
-                                20.000001,
-                                21.0,
-                                100.0});
-
-    test_case.add_expected_output<float>(Shape{4, 4},
-                                         {10.0,
-                                          20.0,
-                                          10.0,
-                                          20.0,
-                                          10.0,
-                                          10.0,
-                                          10.0,
-                                          10.0,
-                                          10.0,
-                                          10.000001,
-                                          15.0,
-                                          19.999999,
-                                          20.0,
-                                          20.0,
-                                          20.0,
-                                          20.0});
-
-    test_case.run();
+namespace
+{
+    template <typename T>
+    void clamp_test(const string& backend,
+                    const element::Type& type,
+                    const PartialShape& dynamic_shape,
+                    const Shape& static_shape,
+                    const std::vector<T>& input,
+                    double min,
+                    double max,
+                    const std::vector<T>& output)
+    {
+        auto data = make_shared<op::Parameter>(type, dynamic_shape);
+        auto clamp = make_shared<op::Clamp>(data, min, max);
+        auto function = make_shared<Function>(clamp, ParameterVector{data});
+
+        auto mode = test::BackendMode::STATIC;
+        if (dynamic_shape.is_dynamic())
+        {
+            mode = test::BackendMode::DYNAMIC;
+        }
+        auto test_case = test::NgraphTestCase(function, backend, mode);
+        test_case.add_input<T>(static_shape, input);
+        test_case.add_expected_output<T>(static_shape, output);
+        test_case.run();
+    }
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, fused_clamp_double)
+{
+    auto type = element::f64;
+    typedef double ctype;
+
+    auto sshape = Shape{5, 2};
+    auto dshape = PartialShape::dynamic();
+
+    auto min = numeric_limits<ctype>::min();
+    auto max = numeric_limits<ctype>::max();
+    auto pinf = numeric_limits<double>::infinity();
+    auto ninf = -numeric_limits<double>::infinity();
+
+    vector<ctype> input{min, max, ninf, pinf, 9.99999, 10.0, 10.000001, 19.999999, 20.0, 20.000001};
+
+    // static shape
+    clamp_test<ctype>("${BACKEND_NAME}",
```

Comment:
I want to see one test with input 0.1, 0.2, ... 1.0 with a min of 0.2 and a max of 0.8 or something like that. That should work.

Reply:
OK, will add.

Reply:
Done.